docs/summer-of-code.md

```diff
@@ -0,0 +1,55 @@
+# Welcome to Probot's Summer of Code!
+
+In 2018, Probot is participating in Google's Summer of Code and Rails Girls Summer of Code, two amazing programs that help students connect with open source projects.
+
+## FAQ
+
+#### How do I get started with Probot?
+
+We suggest that you build our ["Hello World" app](https://probot.github.io/docs/hello-world/) to get the hang of registering a GitHub App and the base setup of a Probot App.
+
+You can also [join our Slack](https://probot-slackin.herokuapp.com/) and join the **#summer-of-code** channel.
+
+#### Can I claim this cool idea?
+
+We're excited that you're excited, but the Summer of Code ideas in our [ideas repo](https://github.com/probot/ideas) are for the students that are accepted to work on Probot over the summer. Claims to those ideas will be ignored until applicants are officially accepted.
```

Comment:
maybe reword to "Claims to those ideas will be ignored until April (when the submission deadline passed)"